wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -67,16 +68,14 @@ open class LockScreenActivity : SecureActivity() {
 
     @Inject lateinit var baseAlertDialogBuilder: BaseAlertDialogBuilder
     protected lateinit var alertDialog: AlertDialog
-
-    val walletApplication: WalletApplication = WalletApplication.getInstance()
-    private val configuration = walletApplication.configuration
-    private val autoLogout: AutoLogout = walletApplication.autoLogout
-    @Inject
-    lateinit var lockScreenBroadcaster: LockScreenBroadcaster
+    @Inject lateinit var walletApplication: WalletApplication
+    @Inject lateinit var lockScreenBroadcaster: LockScreenBroadcaster
+    @Inject lateinit var configuration: Configuration
+    private val autoLogout: AutoLogout by lazy { walletApplication.autoLogout }
```

Comment:
Injecting these fields instead of calling `getInstance()` allows to run unit tests on derived activities.